src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -309,7 +300,6 @@ public function load(array $configs, ContainerBuilder $container)
                 'Symfony\\Component\\DependencyInjection\\Container',
 
                 'Symfony\\Component\\EventDispatcher\\Event',
-                'Symfony\\Component\\EventDispatcher\\ContainerAwareEventDispatcher',
```

Comment:
I think you should revert this line: it will only create a conflict with that other PR removing the block